### ISSUE TITLE
PEER_PORT should not be 80 for https

### ISSUE THREAD

**Issue by malafeev** [opentracing-contrib/java-apache-httpclient#10]
From https://github.com/opentracing-contrib/java-apache-httpclient/blob/5c7b84beadb27c1dc9f16784e2532acc5be4bc81/opentracing-apache-httpclient/src/main/java/io/opentracing/contrib/apache/http/client/ApacheClientSpanDecorator.java#L63
```
Tags.PEER_PORT.set(span, uri.getPort() == -1 ? 80 : uri.getPort());
```
It sets `80` what is right only for `http`.
But for `https` it should be `443`.

**Comment by wtf182** [comment]
所以需要先判断一下是https还是http

**Comment by jefflnorton** [comment]
I have fixed this as part of #15.
